test/cluster/object_store/test_backup.py

```diff
@@ -768,6 +768,71 @@ async def test_restore_with_non_existing_sstable(manager: ManagerClient, object_
     assert 'state' in status and status['state'] == 'failed'
     assert 'error' in status and 'Not Found' in status['error']
 
+
+@pytest.mark.asyncio
+async def test_backup_broken_streaming(manager: ManagerClient, s3_storage):
```

Comment:
Needs to be marked xfail, since at this point, the fix is not yet committed.

This can hit someone who is bisecting, and if bisect lands at this commit, it will generate a false positive.